engine/src/main/resources/assets/i18n/menu.lang

```diff
@@ -50,6 +50,8 @@
     "category-mouse": "category-mouse",
     "category-movement": "category-movement",
     "category-nui": "category-nui",
+    "change-popup-message": "change-popup-message",
```

Comment:
Noted!

I'd rather not overcomplicate things re: Weblate setup unless we're going for a fully reworked system with strings split into 5-6 files.